components/ibm-components/spark/data_preprocess_spark/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.6.8-stretch
+FROM jupyter/pyspark-notebook:c1b0cf6bf4d6
```

Comment:
Any chance we can create a custom image for this?

Reply:
ok, I can try to build our own pyspark image with spark 2.1.

Reply:
Renamed to our own custom build image.